superchain/Dockerfile

```diff
@@ -35,19 +35,39 @@ RUN amazon-linux-extras install ruby2.4
-RUN amazon-linux-extras install corretto8                                                                               \
-  && yum -y install maven                                                                                               \
+# Install JDK8 (Corretto)
+RUN amazon-linux-extras enable corretto8                                                                                \
+  && yum -y install java-1.8.0-amazon-corretto-devel                                                                    \
   && yum clean all && rm -rf /var/cache/yum
+
+# Install Maven
+ENV M2_VERSION 3.6.3
+RUN curl -sL https://www.apache.org/dist/maven/maven-3/${M2_VERSION}/binaries/apache-maven-${M2_VERSION}-bin.tar.gz     \
+         -o /tmp/apache-maven.tar.gz                                                                                    \
+  && curl -sL https://www.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz.asc                \
+          -o /tmp/apache-maven.tar.gz.asc                                                                               \
+  && mkdir -p /tmp/gpg-maven && chmod go-rwx /tmp/gpg-maven                                                             \
+  && curl -sL https://www.apache.org/dist/maven/KEYS | gpg --homedir /tmp/gpg-maven --import                            \
+  && gpg --homedir /tmp/gpg-maven --verify /tmp/apache-maven.tar.gz.asc /tmp/apache-maven.tar.gz                        \
+  && mkdir -p /usr/local && (cd /usr/local && tar xzf /tmp/apache-maven.tar.gz)                                         \
+  && mv /usr/local/apache-maven-${M2_VERSION} /usr/local/apache-maven                                                   \
+  && for BIN in $(find /usr/local/apache-maven/bin -type f -executable);                                                \
+     do                                                                                                                 \
+       ln -s $BIN /usr/local/bin/$(basename $BIN);                                                                      \
+     done                                                                                                               \
+  && rm -rf /tmp/apache-maven.tar.gz /tmp/apache-maven.tar.gz.asc /tmp/gpg-maven
 COPY m2-settings.xml /root/.m2/settings.xml
+ENV M2_HOME=/usr/local/apache-maven                                                                                     \
+    M2=/usr/local/apache-maven/bin                                                                                      \
+    MAVEN_OPTS="-Xms256m -Xmx512m"
 
 # Install Docker
 RUN amazon-linux-extras install docker                                                                                  \
   && yum clean all && rm -rf /var/cache/yum
 VOLUME /var/lib/docker
 
 # Install shared dependencies
-RUN yum -y install awscli git gzip openssl rsync tar unzip which zip                                                    \
```

Comment:
Why was “tar” removed? Sounds like a useful thing to have in the image.

Reply:
`tar` wasn't removed, it was moved upfront because it's needed before we get to this point (to install `maven` from tarball, of course 😁)